### PR TITLE
feat: implement styles to the playground section

### PR DESCRIPTION
**Pull request by gui-santos**
# Purpose of PR

Screenshot 2022-01-25 at 14 32 03

- Simplifies the playground page rendering which means no more "getLayout" method, we use the power of CSS to show or hide stuff on the screen
- Adds some alignment styles to the playground page
- Removes the footer from the playground page
- Uses "embedded" buttons in the playground topbar like the ones we see in Compose and Launch

WDYT?

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

**Review by domarku — APPROVED** [review]
J'APPROVE

File: packages/website/components/Playground/Playground.tsx [hunk 1]
```diff
@@ -42,20 +32,12 @@ export default function Playground() {
...
-      {code && <SandpackRenderer code={code} topbar={<PlaygroundTopBar />} />}
-    </div>
-  );
-}
-
-const defaultCode = `
-import { Button } from '@contentful/f36-components';
+  if (!code) {
+    return null;
+  }
```
Comment (bgutsol) [inline packages/website/components/Playground/Playground.tsx:40]: You might not need this check, `code` will always be a string

**Review by bgutsol — APPROVED** [review]
Nice refactoring 💎

Only one minor comment

File: packages/website/components/Playground/Playground.tsx (hunk 1 above)
Comment (gui-santos) [reply packages/website/components/Playground/Playground.tsx:40]: If I remove it the whole thing explodes

Screenshot 2022-01-25 at 16 48 24

**Pull request merged**